iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -155,7 +155,13 @@ export class Inspector {
 
     handleMessage(event) {
         const jsonString = event.originalEvent.data;
-        const data = JSON.parse(jsonString);
+        let data;
+        try {
+            data = JSON.parse(jsonString);
+        }
+        catch (e) {
+            return;
```

Comment:
Out of curiosity, why is this needed?  Does write-excel-file send messages internally to itself?

Reply:
Seemingly - something is creating 20 or so messages at the point that the file is downloaded.  I couldn't immediately make sense of it, but I think it might be coming from `jsZip` which `write-excel-file` depends on.

Reply:
It would be good to add a comment explaining that we're swallowing exceptions in order to ignore messages coming from the `write-excel-file` dependency.